app/views/spree/shared/_search.html.erb

```diff
@@ -1,15 +1,25 @@
-<%= form_tag spree.products_path, method: :get, class: "py-md-1 d-flex align-items-center search-form" do %>
-  <%= search_field_tag :keywords,
+<%#= form_tag spree.products_path, method: :get, class: "py-md-1 d-flex align-items-center search-form" do %>
+  <%#= search_field_tag :keywords,
                        params[:keywords],
                        placeholder: Spree.t(:search),
                        title: Spree.t(:search),
                        autofocus: true,
                        'aria-label': Spree.t(:search),
                        class: " border-0 p-3" %>
-  <button type="submit" class="btn border-0 p-0" aria-label="Submit">
-    <%= icon(name: 'search',
+  <!-- <button type="submit" class="btn border-0 p-0" aria-label="Submit">
+    <%#= icon(name: 'search',
             classes: 'd-inline',
             width: 27,
             height: 27) %>
-  </button>
+  </button> -->
+<%# end %>
+<%= form_tag spree.products_path, method: :get , id:'global-search-button' do%>
+  <div class="header-search tablet-hide" id="search-dropdown">
+    <span class="search-custom-btn"></span>
+    <%= search_field_tag :keywords,
```

Comment:
You missed these options from search_field. As its present in above default spree

autofocus: true,
'aria-label': Spree.t(:search),